### PR TITLE
docs: Update the documentation for Card and CardContentControl

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## Description

Update the documentation for Card and CardContentControl

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [X] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: doc/controls/CardAndCardContentControl.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Card & CardContentControl
+
+## Summary
+
+Card represents a control identifiable as a single, contained unit used to visually group related child content and actions that relate information about a subject.
+A card’s layout and dimensions depend on its contents.
```
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:6]: french apostrophe
```suggestion
A card's layout and dimensions depend on its contents.
```

File: doc/controls/CardAndCardContentControl.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+## Remarks
+
+Currently, there are three [Material](https://m3.material.io/components/cards/) styles for `Card` and `CardContentControl` that you can use. 
+Depending on the amount of user attention you want to draw to the content you can use:
+- `ElevatedCardStyle` or `ElevatedCardContentControlStyle` to add a subtle z-axis elevation.
+- `FilledCardStyle` or `FilledCardContentControlStyle` to display a simple background color without any elevation or border for the card.
+- `OutlinedCardStyle` or `OutlinedCardContentControlStyle` to display a simple solid stroke along the border of the card.
+
+## Card
+The Card control comes with all the build-in properties deriving from `Control` but also has these additional properties listed below that you can use depending on the content that you want to display:
```
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:17]:
```suggestion
The Card control comes with all the built-in properties of a `Control`  and also a few additional properties listed below that let you tailor the content to be displayed:
```

File: doc/controls/CardAndCardContentControl.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+| Property                  | Type              |
+|---------------------------|-------------------|
+| HeaderContent             | Object            |
```
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:21]: prefer the lowercase alias
```suggestion
| HeaderContent             | object            |
```
same for `double` below

File: doc/controls/CardAndCardContentControl.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+| MediaContent              | Object            |
+| MediaContentTemplate      | DataTemplate      |
+| SupportingContent         | Object            |
+| SupportingContentTemplate | DataTemplate      |
+| IconsContent              | Object            |
+| IconsContentTemplate      | DataTemplate      |
+| Elevation                 | Double            |
+| ShadowColor               | Windows.UI.Color  |
+
+If you want to have something more custom for the entire content you can use  [CardContentControl](#cardcontentcontrol) instead of using `Card`.
```
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:36]:
```suggestion
Consider using [CardContentControl](#cardcontentcontrol) if you need full control over the content layout.
```

File: doc/controls/CardAndCardContentControl.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+### Usage for Card
+
+#### ElevatedCardStyle
```
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:40]: you can merge the 3 example into a single one
and replace the #### headers with \<-- xml comments -->

File: doc/controls/CardAndCardContentControl.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+| SupportingContent         | Object            |
+| SupportingContentTemplate | DataTemplate      |
+| IconsContent              | Object            |
+| IconsContentTemplate      | DataTemplate      |
+| Elevation                 | Double            |
+| ShadowColor               | Windows.UI.Color  |
+
+If you want to have something more custom for the entire content you can use  [CardContentControl](#cardcontentcontrol) instead of using `Card`.
+
+### Usage for Card
```
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:38]: we are already nesting under ## Card
```suggestion
### Usage
```

File: doc/controls/CardAndCardContentControl.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+<utu:Card HeaderContent="Outlined card"
+		  SubHeaderContent="With title and subtitle"
+		  Style="{StaticResource OutlinedCardStyle}" />
+```
+
+## CardContentControl
+
+The CardContentControl is based on ContentControl and allow you to custom the entire content to fit your needs inside the DataTemplate.
```
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:71]:
```suggestion
The `CardContentControl` is based on `ContentControl` and allows you to customize the entire content through `DataTemplate` to fit your needs.
```

File: doc/controls/CardAndCardContentControl.md (hunk 5 above)
Comment (Xiaoy312) [reply doc/controls/CardAndCardContentControl.md:40]: same for cardcontentcontrol below

File: doc/controls/CardAndCardContentControl.md (hunk 6 above)
Comment (Xiaoy312) [reply doc/controls/CardAndCardContentControl.md:38]: same for cardcontentcontrol below

File: doc/controls/CardAndCardContentControl.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
+# Card & CardContentControl
+
+## Summary
```
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:3]: also worth adding an one-liner for the CardContentControl in the summary

File: doc/controls/CardAndCardContentControl.md (hunk 1 above)
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:6]: maybe move this line under ##card?

File: doc/controls/CardAndCardContentControl.md (hunk 4 above)
Comment (Xiaoy312) [reply doc/controls/CardAndCardContentControl.md:36]: "the layout of the content"? not sure which flows better

File: doc/controls/CardAndCardContentControl.md (hunk 1 above)
Comment (agneszitte-nventive) [reply doc/controls/CardAndCardContentControl.md]: I decided to change the content to do a summary for both at the same time as they have the same definition at the end

File: doc/controls/CardAndCardContentControl.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+Consider using [CardContentControl](#cardcontentcontrol) if you need full control over the content layout.
+
+### Usage
+
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<-- ElevatedCardStyle -->
```
Comment (Xiaoy312) [inline doc/controls/CardAndCardContentControl.md:44]:
```suggestion
<!-- ElevatedCardStyle -->
```
also can remove the blank line between comment and the <utu:Card />

Comment (Xiaoy312) [reply doc/controls/CardAndCardContentControl.md:44]: same for all occurrences

File: doc/toc.yml [hunk 10]
```diff
@@ -7,6 +7,6 @@ items:
 - name: Controls
   items:
     - name: CardContentControl
```
Comment (Xiaoy312) [inline doc/toc.yml:9]: maybe change both title and file name to "Cards"?
```suggestion
    - name: Cards
```

File: doc/controls/CardAndCardContentControl.md (hunk 9 above)
Comment (agneszitte-nventive) [reply doc/controls/CardAndCardContentControl.md:44]: good catch thanks ;)

File: doc/toc.yml (hunk 10 above)
Comment (agneszitte-nventive) [reply doc/toc.yml]: good catch thanks !

**Review by Xiaoy312 — APPROVED** [review]
noice~

**Review by kazo0 — APPROVED** [review]
🎉

File: doc/controls/CardAndCardContentControl.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
+# Card & CardContentControl
+
+## Summary
```
Comment (jeromelaban) [inline doc/controls/CardAndCardContentControl.md:3]: Can we add screenshots?

Comment (agneszitte-nventive) [reply doc/controls/CardAndCardContentControl.md:3]: Yes @jeromelaban but we will add them as the same time that the v2 styles will to be done for Card next week. (cc @kazo0)

**Pull request merged**